x-pack/legacy/plugins/alerting/server/routes/update_api_key.test.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { createMockServer } from './_mock_server';
+import { updateApiKeyRoute } from './update_api_key';
+
+const { server, alertsClient } = createMockServer();
+updateApiKeyRoute(server);
+
```

Comment:
If we do want it to be an error to call this API on a disabled alert, there should be a test for that.

Reply:
Agreed. Since this file is only making sure `alertsClient` is called with proper arguments. The test lives within the alerts client. Those files are pretty big so the tests are a bit hidden.

Unit test: https://github.com/elastic/kibana/pull/45925/files#diff-dda4aa696499278b342e4b1d4590925dR1269

API integration test: https://github.com/elastic/kibana/pull/45925/files#diff-a8e40fcc59d37bca042ea0f1854edb69R71.